### PR TITLE
Add addtoken in  shoprepo

### PR DESCRIPTION
**Pull request by bavanya**

### CODE REVIEW

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/ShopController.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin
+public class ShopController {
```
Comment (KarishmaSinghChauhan) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/ShopController.java:31]: Compare with Ishita's changes. This has been renamed I think.

Comment (stringtheory1234) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/ShopController.java:31]: Yeah rename to VirtualQueueController

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/ShopService.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+import com.google.gpay.virtualqueue.backendservice.proto.CreateShopRequest;
+import com.google.gpay.virtualqueue.backendservice.proto.CreateShopResponse;
+import com.google.gpay.virtualqueue.backendservice.repository.VirtualQueueRepository;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ShopService {
```
Comment (stringtheory1234) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/ShopService.java:29]: rename to VirtualQueueService

Comment (KarishmaSinghChauhan) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/ShopService.java:29]: Same

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/ShopController.java (hunk 1 above)
Comment (bavanya) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/ShopController.java]: Done

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/ShopService.java (hunk 2 above)
Comment (bavanya) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/ShopService.java]: Done

Comment (bavanya) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/service/ShopService.java]: Done

**Comment by bavanya** [comment]
> Resolve the merge conflicts.
Resolved them.

**Pull request closed without merging**